client/src/components/ProductDetails/Details.js

```diff
@@ -33,15 +33,21 @@ const Details = ({ product }) => {
   const classes = DetailsStyles();
   const state = useAuthState();
   const history = useHistory();
-  // console.log(state.user);
+  console.log(state);
```

Comment:
Please remove the log if not needed

Reply:
log will be needed to see the state as of now .Because I need to make sure that the state object is available because I have to see what happens when my token expires